recipe/meta.yaml

```diff
@@ -1,34 +1,30 @@
 {% set name = "tslearn" %}
-{% set version = "0.5.3.2" %}
-{% set sha256 = "acf4015b07207ad6a645ddcc8eec04d8431716b24db66a745ab29fa22f772820" %}
+{% set version = "0.6.3" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c855c0b784b42dc4c50f4eaf524ec7ed658037e86fc14af38ca3b66c75dd6adb
 
 build:
   number: 0
-  skip: true  # [py2k]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py>312 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - {{ compiler('c') }}
+    - python
+    - numpy
   host:
     - python
     - pip
-    - cython
-    - numpy
+    - numpy {{ numpy }}
```

Comment:
Could you try to build against numpy 2?
```
    - numpy 2.0 # [py<313]
    - numpy 2.1 # [py>=313]
```

Reply:
There seems to be some Unsatisfiable dependencies when running with numpy 2

```
{MatchSpec("numpy==1.19.5=py38heec5a64_2"), MatchSpec("libblas[version='>=3.9.0,<4.0a0']")}

- package numpy-1.19.5-py38heec5a64_2 requires libblas >=3.9.0,<4.0a0, but none of the providers can be installed
```

Reply:
We haven't built numba yet for numpy 2 (there is a PR opened https://github.com/AnacondaRecipes/numba-feedstock/pull/19). Once it is completed (today hopefully), this should unblock you. 
That being said, this version was released end of 2023, so there is a possibility that the upstream code is not compatible with numpy 2. This you will see by running the tests. If the tests do not pass because of this, you could revert to numpy 1.